### ISSUE TITLE
Anyone else getting Unknowns?

### ISSUE THREAD

**Issue by PYehl73** [briis/hass-weatherflow#80]
lot of my main data that I want to see are showing up as Unknown - but not all data?

![image](https://github.com/briis/hass-weatherflow/assets/119134827/840f212a-c5fd-457b-9f82-3bbce2a85a34)

**Comment by jasii** [comment]
Yes, I just started getting an error with my setup and unknowns.

**Comment by UpDryTwist** [comment]
See my patch for this over on this issue:  #81

**Comment by briis** [comment]
Duplicate of #81
